nebula/ui/components/VPNConnectionInfoContent.qml

```diff
@@ -132,9 +133,9 @@ VPNFlickable {
                         }
 
                         VPNConnectionInfoItem {
-                            title: VPNServerCountryModel.getLocalizedCountryName(
-                                VPNCurrentServer.exitCountryCode
-                            )
+                            title: serverLocations.isMultipHop
+                            ? VPNCurrentServer.localizedCityName
+                            : VPNServerCountryModel.getLocalizedCountryName(VPNCurrentServer.exitCountryCode);
```

Comment:
This is the only place where we use: `VPNServerCountryModel.getLocalizedCountryName`.
Can we get rid of it? I would like to see: `VPNCurrentServer.localizedCountryName` instead.
Do you mind to do this? As part of this PR or a separate one.

Reply:
Jira issue: https://mozilla-hub.atlassian.net/browse/VPN-3884